index.html

```diff
@@ -22,5 +22,29 @@
         </div>
       </nav>
     </header>
+    <main class="main">
+      <h1 class="h1">I'M Charle Peter. Glad to see you!</h1>
+      <p class="p">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h4 class="h4-connect">LET'S CONNECT</h4>
+      <div class="flex item-center">
+        <a class="icon-tag" href="#"
+          ><img src="/image/twitter.png" alt="twitter"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/linkedin.png" alt="linkedin"
+        /></a>
+        <a class="icon-tag" href="#"><img src="/image/m.png" alt="m" /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/git.png" alt="Github"
+        /></a>
+        <a class="icon-tag" href="#"
+          ><img src="/image/angelist.png" alt="angelist"
+        /></a>
+      </div>
```

Comment:
- [ ] kindly update your code to make it more semantic, the social icons should be listed inside a list `<ul> & <li>` instead of normal divs, and each social icon should be an `<a>` that contains the link for that icon
example:
```HTML
<ul class='...'>
<li class='...'><a href='...' ><img>...</img></a></li>
<li class='...'><a href='...' ><img>...</img></a></li>
...
<li class='...'><a href='...' ><img>...</img></a></li>
</ul>
```